apps/web/src/components/FilterDialog/FilterDialog.tsx

```diff
@@ -29,13 +29,16 @@ type FilterDialogProps<TFieldValues extends FieldValues> = {
   resetValues: CommonFilterDialogProps<TFieldValues>["resetValues"];
   defaultOpen?: boolean;
   children: React.ReactNode;
+  /** Modify the filter count in the button (most commonly used for hidden filters) */
+  modifyFilterCount?: number;
```

Comment:
Addition with negative integers also makes sense to me, I think this also looks understandable how to use it.